doc/changelog/08-vernac-commands-and-options/17544-clearbody-att.rst

```diff
@@ -0,0 +1,10 @@
+- **Added:**
+  :attr:`clearbody` for :cmd:`Let` providing the behavior of using :cmd:`Qed`
+  (clearing the body of such letins in interactive proofs without kernel enforcement)
+  in a more explicit way
```

Comment:
```suggestion
  :attr:`clearbody` for :cmd:`Let` to clear the body of a let-in in an interactive
  proof without kernel enforcement.  (This is the behavior that was previously
  provided by using :cmd:`Qed`, which is now deprecated for `Let`\s.)
```

I'm not quite sure what "without kernel enforcement" means/implies--is it common knowledge or worth spelling out a little more?

Has it been significantly more common to use `Defined` or `Qed` to end these proofs?  If it's been `Qed` by a significant margin, perhaps the attribute should be  `keepbody` and `Defined` should be deprecated so the default is what is most common.  (Though if users would be surprised by the body being cleared, that would argue for doing it as you've proposed.)